### DISCUSSION TITLE
Rendu final

### DISCUSSION THREAD

**Discussion by TitusVM** [HE-Arc/HabiTrack#42] [Announcements]
This is the official first release of the Habitrack App!

<hr /><em>This discussion was created from the release <a href='https://github.com/HE-Arc/HabiTrack/releases/tag/v1.0.0'>Rendu final</a>.</em>

**Comment by TitusVM** [comment]
Wow Congrats! Insane App

**Reply by WindRider97** [reply]
We did it boys
